### DISCUSSION TITLE
Cilium Network Policy for Cilium + Hubble

### DISCUSSION THREAD

**Discussion by davejab** [cilium/cilium#35627] [Q&A]
Hi all 👋 

Lets say, hypothetically, I have default deny on all ingress/egress across the cluster. What would a policy to enable Cilium/Hubble include? I've been trialing this for sometime and can only get Hubble UI to show with no traffic flows, where I would expect at the very minimum to see flows to Hubble UI. Has anyone else managed this? Is this even possible? Any guidance on this would be appreciated.

Thanks

**Comment by joestringer** [comment]
I'm not sure if there's currently an example policy in the tree for this, but perhaps you could use a combination of https://editor.networkpolicy.io/ and the [Firewall Rules](https://docs.cilium.io/en/stable/operations/system_requirements/#firewall-rules) documentation to craft a policy?

There's also a guide on crafting network policies here which you may find relevant: https://docs.cilium.io/en/stable/security/policy-creation/#policy-verdicts
